python/py-pew/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup			python 1.0
+PortGroup           github 1.0
+
+set package			pew
+set user			berdario
+version				1.1.0
+
+name				py-$package
+github.setup        $user $package $version
+github.tarball_from releases
+
+master_sites        ${github.homepage}/archive/
+distname            ${version}
+worksrcdir          ${github.project}-${version}
+
+categories          python
+platforms           darwin
+license             MIT
+
+python.versions		27 35 36
+
+maintainers         openmaintainer @esafak
```

Comment:
Ideally you would also include your email here in obfuscated form:
```
maintainers {domain.com:name @githubhandle} openmaintainer
```

Reply:
I removed py27 support because it apparently needed some other back ports (shutil.which) that do not exist on macports.